index.js

```diff
@@ -901,6 +902,42 @@ class Hyperdrive extends Nanoresource {
     }
   }
 
+  mirror () {
+    const self = this
+    if (this._mirrorRanges) return unmirror
```

Comment:
if you call mirror twice and call the 2nd unmirror, then the event listeneners attached from the first are never cleaned up

Reply:
instead of storing mirrorRanges as state, mb store unmirror itself to simplify that?

Reply:
Fixed